### PR TITLE
Meeting summary for 2016-04-28

### PR DESCRIPTION
**Pull request by G1lius**
Couldn't find the report of the CTAES review. Would be nice to link it.

### CODE REVIEW

**Comment by btcdrak** [comment]
@G1lius It's not been published yet.

File: _posts/en/meetings/2016-04-28-meeting.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+permalink: /en/meetings/2016/04/28/
+name: 2016-04-28-meeting
+type: meetings
+layout: page
+lang: en
+version: 1
+---
+{% include _toc.html %}
+
+- [Link to this week logs](http://bitcoinstats.com/irc/bitcoin-core-dev/logs/2016/04/28#l1461877211.0)
```
Comment (btcdrak) [inline _posts/en/meetings/2016-04-28-meeting.md]: I'm noticing that bitcoinstats.com is regularly down and missing messages. Please can you switch to the botbot.me URL? https://botbot.me/freenode/bitcoin-core-dev/ or link to the meetingbot dump linked in the minutes summary http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-04-28-19.00.log.html

**Comment by G1lius** [comment]
ok, I'll just include a link in a future meeting, or update this one then.
Op 2-mei-2016 14:20 schreef "฿tcDrak" <notifications@github.com>:

**Comment by btcdrak** [comment]
just fyi, search #startmeeting and you can get hyperlinks on botbot https://botbot.me/freenode/bitcoin-core-dev/2016-04-28/?msg=65090086&page=3

**Pull request merged**